ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-ocm-2.3.yaml

```diff
@@ -98,10 +98,9 @@ tests:
       ASSISTED_CONFIG: |
         SERVICE_BASE_REF=ocm-2.3
       POST_INSTALL_COMMANDS: |
-        export TEST=./discovery-infra/tests/test_kube_api.py
         export TEST_FUNC=test_kube_api_ipv4
         export KUBECONFIG=$(find ${KUBECONFIG} -type f)
-        make deploy_assisted_operator test_parallel
+        make deploy_assisted_operator test_kube_api_parallel
```

Comment:
how? you didn't backport the test-infra PR
same for 2.4

Reply:
Right. Removed 2.3, v1, and 2.4